packages/atomic/src/components/commerce/facets/atomic-commerce-facets/atomic-commerce-facets.css.ts

```diff
@@ -0,0 +1,6 @@
+// This file is generated by postcss-build.cjs
```

Comment:
Is the idea to generate these once as we migrate to this new method and then use twind and modify the css.ts files by hand? 

Or will we have to run postcss-build.cjs (hopefully automatically) whenever we make change to the pcss files?